test/common/deployment_types.go

```diff
@@ -13,17 +16,14 @@ import (
 	k8sError "k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/labels"
 
-	integreatlyv1alpha1 "github.com/integr8ly/integreatly-operator/apis/v1alpha1"
-	"github.com/integr8ly/integreatly-operator/pkg/config"
-	"github.com/integr8ly/integreatly-operator/pkg/resources/constants"
-	appsv1 "github.com/openshift/api/apps/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	k8sclient "sigs.k8s.io/controller-runtime/pkg/client"
 )
 
 var (
 	commonApiDeploymentsList = []string{
 		"threeScaleDeployment",
+		"threeScaleProductDeployment",
 		"cloudResourceOperatorDeployment",
 		"observabilityDeployment",
```

Comment:
@trepel just fyi, the observability deployment does not have it's representation in the getDeploymentConfiguration - meaning we are not testing it.